modules/ossm-deploy-single-tenant.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deploy-mod-v2x.adoc
+
+[id="ossm-deploy-single-tenant_{context}"]
+= Single tenancy model
```

Comment:
Single tenancy model -> Single tenancy deployment model ?

Reply:
Yes (this is why I need reviewers!)